core/src/scene/sceneLoader.cpp

```diff
@@ -560,7 +560,7 @@ std::shared_ptr<Texture> SceneLoader::fetchTexture(const std::string& name, cons
                 auto ptr = (unsigned char*)(rawData.data());
                 size_t dataSize = rawData.size();
                 std::lock_guard<std::mutex> lock(m_textureMutex);
-				auto texture = scene->getTexture(name);
+                                auto texture = scene->getTexture(name);
```

Comment:
Bubble Wrap v4.1.0 does use the proper anchor fallbacks now, but that
version is not yet used in Eraser Map the app (it’s pegged to v3.x).

On Wed, Aug 10, 2016 at 2:26 PM, Hannes Janetzek <notifications@github.com>
wrote:

> In core/src/scene/sceneLoader.cpp
> <https://github.com/tangrams/tangram-es/pull/893#discussion_r74334094>:
>
> > @@ -560,7 +560,7 @@ std::shared_ptr<Texture> SceneLoader::fetchTexture(const std::string& name, cons
> >                  auto ptr = (unsigned char*)(rawData.data());
> >                  size_t dataSize = rawData.size();
> >                  std::lock_guard<std::mutex> lock(m_textureMutex);
> > -				auto texture = scene->getTexture(name);
> > +                                auto texture = scene->getTexture(name);
>
> M-x whitespace-cleanup did replace tabs with spaces. @karimnaaji
> <https://github.com/karimnaaji> fix your editor! :p
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/tangrams/tangram-es/pull/893/files/562f172b3d217c7fcbdd754bb5d0102a0e681b0c#r74334094>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA0EO7xuIFHvnlY_QmQnmWk2qgWBb8ovks5qekIKgaJpZM4Ja2Xw>
> .
>